.github/workflows/ci.yml

```diff
@@ -54,3 +27,26 @@ jobs:
         run: cargo clippy --all-features --all-targets
       - name: Rust Test
         run: cargo test --workspace --all-features
+
+  bench-codspeed:
+    name: Benchmark with Codspeed
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ./.github/actions/setup-rust
+
+      - name: Install Codspeed
+        shell: bash
+        run: cargo install --force cargo-codspeed --locked
+
+      - name: Build benchmarks
+        env:
+          RUSTFLAGS: "-C target-cpu=native"
+        run: |
+          cargo codspeed build --profile bench --features test-harness
```

Comment:
Do we use ` --features test-harness` anywhere in this repo?

Reply:
Next PR